extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -277,6 +289,13 @@ export class ShareWidget {
       e.preventDefault();
       this.copyUrlToClipboard_();
     });
+    listen(linkShareButton, 'keyup', (e) => {
+      const code = e.charCode || e.keyCode;
+      if (code === 32 || code === 13) {
```

Comment:
nit:
I wonder if this needs a comment for what keys these are for quick readability.
I think we commonly do top level CONSTs but maybe that's overkill for this.

Reply:
Sounds good to add a comment for this, it's not immediately obvious what these are